setup.py

```diff
@@ -53,12 +53,6 @@ def read_req_file(req_type):
         "Intended Audience :: Science/Research",
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3",
```

Comment:
Is it intentional to drop `Python :: 3`?

Reply:
ahh, no. good catch